### PR TITLE
Network display/change on lock screen

### LINKED ISSUE #1089: You should not have to login to change the network id
Example given:
- User deploys contract in Remix
- Metamask opens
- User discovers he is on mainnet but wants to use testnet.
- User has to login to mainnet 
- then change to testnet
- then deploy contract
- then login on testnet

Should:

- User changes nets, without first login ( changing a net doesn't require login)

Rationale: while converting the transaction pending to suite another net is certainly asking too much, loggin-in to only changing the net is asking too much to the poor developer.   
I understand that the normal user doesn't need this feature, but a good part currently must be developers...

### PR DESCRIPTION
**Pull request by 2-am-zzz**
Fixes #1089

### CODE REVIEW

**Comment by Zanibas** [comment]
screenshot from 2017-02-15 15-57-16

**Comment by Zanibas** [comment]
screenshot from 2017-02-15 15-57-52

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 ## Current Master
 - net_version has been made synchronous.
 - Test suite for migrations expanded.
+- Network now changeable from lock screen.
+
```
Comment (flyswatter) [inline CHANGELOG.md]: remove blank line

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
 ## Current Master
 - net_version has been made synchronous.
```
Comment (flyswatter) [inline CHANGELOG.md]: Add blank line after current master heading.

File: ui/app/app.js [hunk 3]
```diff
@@ -190,7 +190,7 @@ App.prototype.renderAppBar = function () {
           ]),
 
           // hamburger
-          h(SandwichExpando, {
+          props.isUnlocked && h(SandwichExpando, {
```
Comment (flyswatter) [inline ui/app/app.js]: I kindof wish there was a way to hide these things without calling `props.isUnlocked &&` before each element.

Surely making the menu bar its own component could offer us a better abstraction.

**Review by danfinlay — APPROVED** [review]

**Comment by flyswatter** [comment]
Okay, there are some optimizations that could have been made, but this is solid and useful for now.

**Pull request merged**